cogs/birthday.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import re
+from typing import Pattern
+
+from discord.ext import commands
+
+
+class Birthdays(commands.Cog):
+    DATEPATTERN: Pattern[str] = re.compile(r'(((0[1-9])|(1[0-9])|(2[0-9])|(3[0-1]))\.'  # 01.-31.
```

Comment:
I feel like we could simplify this a lot...

Reply:
I updated the RegEx.. it should be faster now.. no idea how to simplify this elsewise